webfront/tests/fixtures_reader.py

```diff
@@ -161,7 +163,7 @@ def get_fixtures(self):
                     c["structure_chain_acc"] = sp["chain"]
                     c["text_structure"] = c["structure_acc"] + " " + sp["chain"]
 
-                    c["entry_structure_locations"]= ep["coordinates"],
+                    c["entry_structure_locations"] = (ep["coordinates"],)
```

Comment:
```suggestion
                    c["entry_structure_locations"] = ep["coordinates"]
```

Wrapping in parentheses makes a tuple, resulting in the following:

```json
{
    "entry_structure_locations": [
        [
            {
                "fragments": [
                    {
                        "start": 10,
                        "end": 48
                    }
                ]
            }
        ]
    ]
}
```

instead of 

```json
{
    "entry_structure_locations": [
        {
            "fragments": [
                {
                    "start": 10,
                    "end": 48
                }
            ]
        }
    ]
}
```

Reply:
I think that was `black` reformatting 😱